example/styleguide/0-Getting Started/1-Installation & Usage.md

```diff
@@ -60,12 +60,12 @@ You can also show component code in your MDX files using the traditional backtic
 
 #### Other MDX shortcodes available:
 
-`<DarkWrapper>`: Wraps the contents of this component in a "dark" wrapper (opposite of background color). Usage:
+`<ContrastWrapper>`: Wraps the contents of this component in a "dark" wrapper (opposite of background color). Usage:
```

Comment:
Same question about the word "dark" here

Reply:
Yep